deepcell_tracking/tracking_test.py

```diff
@@ -71,10 +71,10 @@ def _get_dummy_tracking_data(length=128, frames=3,
 class DummyModel(object):  # pylint: disable=useless-object-inheritance
 
     def predict(self, data):
-        if isinstance(data, list):
-            batches = 0 if not data else len(data[0])
+        if isinstance(data, dict):
+            batches = 0 if not data else next(iter(data.values())).shape[0]
         else:
-            batches = len(data)
+            batches = next(iter(data.values())).shape[0]
```

Comment:
Could you write a brief comment here explaining what's going on?

Reply:
The `DummyModel` used to be able to check input data to identify how many "cells" were being compared by accessing the first element of the input list and passing that to the output. Because the data is no longer in a list, this no longer works. Instead, we take a random value from the dictionary and select the first dimension (the batch dimension) to identify how many "cells" are being compared.